test/integration/apiserver/discovery/discovery_test.go

```diff
@@ -52,9 +52,16 @@ import (
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
+//lint:ignore U1000 we need to alias only for the sake of embedding since all are called Interface
 type kubeClientSet = kubernetes.Interface
+
+//lint:ignore U1000 we need to alias only for the sake of embedding since all are called Interface
 type aggegatorClientSet = aggregator.Interface
+
+//lint:ignore U1000 we need to alias only for the sake of embedding since all are called Interface
 type apiextensionsClientSet = apiextensions.Interface
+
+//lint:ignore U1000 we need to alias only for the sake of embedding since all are called Interface
```

Comment:
Fwiw - I'm not a fan of doing this.

The warning we get is that these are unused, one way to mitigate this is to do a workaround like:
```go
type testClientSet struct {
 	kubeClientSet
	aggegatorClientSet
	apiextensionsClientSet
	dynamicClientset
    _ kubeClientSet
    _ aggegatorClientSet
    _ apiextensionsClientSet
    _ dynamicClientset
}
```
I'm not sure that's the best way to go about it.

The other way to mitigate it would be to not embed the interfaces but to have a field for each client and then write a fake client, could do that if its preferred! I went ahead with this since it seemed like the cleanest way to go about it.

Reply:
the embedding with type aliasing is slightly weird, but not wrong

I'd rather make the smallest change here (ignoring the false positive) and report the issue to the linter